docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source

```diff
@@ -0,0 +1,109 @@
+---
+id: aws-security-data-lake-source
+title: AWS Security Data Lake Source (Beta)
+description: This document explains how to configure the AWS Security Data Lake source setup using the Sumo Logic environment.
+---
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+AWS Security Data Lake source provides a web services interface that can be used to manage data from multiple AWS sources through a single interface for multiple AWS organizations.
+
+This makes AWS log setup relatively simple. Furthermore, the data available from various AWS services is in the OCSF format, making it simple to parse and correlate across different sources.
+
+
+## Configure AWS Security Data Lake Setup
+
+![Step 1. Grant Sumo Logic access to AWS Security Data Lake Console](#step-1-grant-sumo-logic-access-to-aws-security-data-lake-console)
+![Step 2. Setup Subscriber in AWS Security Data Lake Console](#step-2-setup-subscriber-in-aws-security-data-lake-console)
+![Step 3. Create an AWS Security Data Lake Source using Sumo Logic Environment](#step-3-create-an-aws-security-data-lake-source-using-sumo-logic-environment)
+![Step 4. Update Subscriber](#step-4-update-aws-security-data-lake-subscriber)
```

Comment:
This table of contents already appears in the right nav. It'd be redundant to have it twice.
```suggestion

```